### PR TITLE
Feature/gql campaign owner display name

### PR DESCRIPTION
**Pull request by johantjellden**
We need to be able to query out who the campaign owner is of a
campaign. For referral campaigns this needs to be set to null in
order to not leak data about our members.

### CODE REVIEW

File: src/main/java/com/hedvig/productPricing/campaign/graphql/types/Campaign.kt [hunk 1]
```diff
@@ -3,17 +3,18 @@ package com.hedvig.productPricing.campaign.graphql.types
...
+  val owner: CampaignOwner? = null
 ) {
 
   val code: String
     get() = _code.toUpperCase()
 
   companion object {
-    fun of(c: DomainCampaign) = Campaign(c.code, Incentive.of(c.incentive))
-    fun of(c: RedeemedCampaign) = Campaign(c.campaign.code, incentive = Incentive.of(c.incentive))
+    fun of(c: DomainCampaign) = Campaign(c.code, Incentive.of(c.incentive), owner = CampaignOwner.of(c.campaignOwner))
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/productPricing/campaign/graphql/types/Campaign.kt]: I personally dislike variables named by a single letter, e.g. "c", but you decide

File: src/main/java/com/hedvig/productPricing/campaign/domain/model/CampaignOwner.kt [hunk 2]
```diff
@@ -2,10 +2,7 @@ package com.hedvig.productPricing.campaign.domain.model
 
 import org.hibernate.annotations.CreationTimestamp
 import java.time.Instant
-import javax.persistence.Entity
-import javax.persistence.Id
-import javax.persistence.Inheritance
-import javax.persistence.InheritanceType
+import javax.persistence.*
```
Comment (hedvigoscar) [inline src/main/java/com/hedvig/productPricing/campaign/domain/model/CampaignOwner.kt]: Wildcard imports 😮

File: src/main/java/com/hedvig/productPricing/campaign/graphql/types/CampaignOwner.kt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import com.hedvig.productPricing.campaign.domain.model.Partner
+import com.hedvig.productPricing.campaign.domain.model.CampaignOwner as DomainCampaignOwner
+
+data class CampaignOwner (
+  val id: String,
+  val displayName: String)
+{
+  companion object {
+    fun of(co:DomainCampaignOwner?):CampaignOwner? {
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/productPricing/campaign/graphql/types/CampaignOwner.kt]:
```suggestion
    fun of(domainCampaignOwner: DomainCampaignOwner?): CampaignOwner? {
```

File: src/main/java/com/hedvig/productPricing/campaign/domain/model/CampaignOwner.kt (hunk 2 above)
Comment (johantjellden) [reply src/main/java/com/hedvig/productPricing/campaign/domain/model/CampaignOwner.kt]: IntelliJ makes wildcardimports sometimes.

File: src/main/java/com/hedvig/productPricing/campaign/graphql/types/Campaign.kt (hunk 1 above)
Comment (johantjellden) [reply src/main/java/com/hedvig/productPricing/campaign/graphql/types/Campaign.kt]: It depends on context, these are small functions that only take one parameter so I think that it is ok.

**Pull request merged**